YMChat/src/main/java/com/yellowmessenger/ymchat/BotWebView.java

```diff
@@ -177,6 +177,7 @@ public void onCreate(Bundle savedInstanceState) {
                 case "close-bot":
                     closeBot();
                     this.finish();
+                    YMChat.getInstance().emitEvent(new YMBotEventResponse("bot-closed", ""));
```

Comment:
Put this line before calling this.finish(), otherwise it may not get called.

Reply:
Done